readthedocs/projects/models.py

```diff
@@ -677,6 +723,35 @@ def custom_path_prefix(self):
             return self.urlconf.split("$", 1)[0]
         return None
 
+    @cached_property
+    def subproject_prefix(self):
```

Comment:
Checked in production, and we don't have projects with a custom urlconf to change the `projects` prefix, so we are okay just checking for the urlpattern attribute.